mno_analysis_tools/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.6-slim
+
+# Install Python tools (git + pipenv)
+RUN apt-get update && apt-get install -y git
+RUN pip install pipenv
+
```

Comment:
we also need a memory profiler because I recall one of the scripts was memory hungry. If it becomes a problem the profiler will point us to the code block we need to optimize.
`# Install memory_profiler if this script is run with PROFILE_MEMORY flag
ARG INSTALL_MEMORY_PROFILER="false"
RUN if [ "$INSTALL_MEMORY_PROFILER" = "true" ]; then \
        apt-get update && apt-get install -y gcc && \
        pip install memory_profiler; \
    fi`

Reply:
Updated thanks.